CHANGELOG

```diff
@@ -1,3 +1,24 @@
+18 December 2016: mitmproxy 1.0
+
+    * Mitmproxy is now Python 3 only
+
+    * The config file format is now a single YAML file. In most cases,
+      converting to the new format should be trivial - please see the docs for
+      more information.
+
+    * console: significant UI improvements - including sorting of flows by
+      size, type and url, status bar improvements, much faster indentation for
+      HTTP views, and more.
+
+    * http2: significant improvements, but is temporarily disabled by default
+      due to wide-spread protocol implementation errors on some large website
+
+    * websockets: the protocol implementation is now mature, and is enabled by
```

Comment:
Please capitalise: `WebSocket`
Maybe mention that we have full support via hooks for messages and content manipulation.